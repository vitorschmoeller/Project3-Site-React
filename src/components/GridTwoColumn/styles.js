import styled, { css } from 'styled-components';
import { Title } from '../Heading/styles';

export const Container = styled.div`
  ${({ theme }) => css`
  display: grid;
  grid-template-columns: 1fr 2fr;
  align-items: center;
  gap: ${theme.spacings.large};

  @media ${theme.media.lteMedium} {
    grid-template-columns: 1fr;
    text-align: center;
  }
  ${Title} {

    margin-bottom: ${theme.spacings.xlarge};
  }
  `}


`;

export const TextContainer = styled.div`
  ${({ theme }) => css`


  @media (max-width: 768px) {
      margin-bottom: ${theme.spacings.large};
    }
  `}
`;

export const ImgContainer = styled.div`

`;

export const Image = styled.img``;
