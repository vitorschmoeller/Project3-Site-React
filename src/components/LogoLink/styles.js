import styled, { css } from 'styled-components';

export const Container = styled.a`
  ${() => css`
    display: flex;
    text-decoration: none;
    align-items: center;
    /* inherits diz que vai herdar do component pai */
    color: inherit;

    > img {
      max-height: 3rem;
    }
  `}
`;
