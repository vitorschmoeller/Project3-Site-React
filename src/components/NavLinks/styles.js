import styled, { css } from 'styled-components';

export const Container = styled.nav`
  ${({ theme }) => css`

    display: flex;
    flex-flow: row wrap;
    font-size: ${theme.font.sizes.medium};

    @media ${theme.media.lteMedium}{
      flex-flow: column wrap;
      align-content: center;
    }

    /* @media (max-width: 768px) {
      display: block;
      text-align: center;
    } */
    `}
`;
