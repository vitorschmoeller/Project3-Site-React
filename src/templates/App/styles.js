import styled from 'styled-components';

export const Wrapper = styled.div`
  background-color: ${(props) => props.theme.colors.blue};

  h1 {
    background: red;

  }
`;
