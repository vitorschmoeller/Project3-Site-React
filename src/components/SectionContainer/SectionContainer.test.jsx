import { screen } from '@testing-library/react';
import { renderTheme } from '../../styles/render-theme';
import { SectionContainer } from '.';

describe('<SectionContainer /> ', () => {
  it('Should render content ', () => {
    const { container } = renderTheme(
      <SectionContainer>
        <h1>Children</h1>
      </SectionContainer>,
    );
    expect(
      screen.getByRole('heading', { name: 'Children' }),
    ).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
