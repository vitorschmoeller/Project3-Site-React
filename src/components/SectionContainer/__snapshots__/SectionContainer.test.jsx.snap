// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionContainer />  Should render content  1`] = `
.c0 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 3.2rem;
}

<div>
  <div
    class="c0"
  >
    <div>
      <h1>
        Children
      </h1>
    </div>
  </div>
</div>
`;
